src/mpid/ch4/include/coll_algo_params.h.in

```diff
@@ -8,186 +8,186 @@
 @ch4_shm_coll_params_include@
 /* *INDENT-ON* */
 
-#define MPIDI_CH4_COLL_AUTO_SELECT (-1)
+#define MPIDI_COLL_AUTO_SELECT (-1)
 
-typedef union MPIDI_Barrier_algo_params {
+typedef union MPIDI_COMMON_Barrier_algo_params {
```

Comment:
I don't think adding COMMON here is the right thing to do. Maybe our "generic" prefix is more suitable? I.e. `MPIDIG_`. I'm not sure.

Reply:
Maybe we can just drop this change if it won't cause issues or collisions.

Reply:
@raffenet after the removal CH4 prefix the names for the union on the CH4 level and union which comprises all layers became identical. That is why we have decided to add prefix COMMON. So, unfortunately, we can't just drop this change. I will add prefix ```MPIDIG_``` instead.